src/main/java/createQuiz/CreateQuizController.java

```diff
@@ -1,4 +1,20 @@
 package createQuiz;
 
+import entities.Deck;
+
+import java.util.List;
+
 public class CreateQuizController {
+
+    private final CreateQuizInputBoundary inputBoundary;
+
+    public CreateQuizController(CreateQuizInputBoundary inputBoundary) {
+        this.inputBoundary = inputBoundary;
+    }
+
+    public CreateQuizOutputData createQuiz(String quizName, List<Deck> quizDecks) {
```

Comment:
Implementation Change: Pass along a list of deckNames and not deck objects to adhere to clean architecture.